simde/x86/avx512vl.h

```diff
@@ -352,6 +352,110 @@ simde_mm256_maskz_abs_epi64(simde__mmask8 k, simde__m256i a) {
 #define _mm256_maskz_abs_epi64(k, a) simde_mm256_maskz_abs_epi64(k, a)
 #endif
 
+SIMDE_FUNCTION_ATTRIBUTES
+simde__m128i
+simde_mm_mask_add_epi8(simde__m128i src, simde__mmask16 k, simde__m128i a, simde__m128i b) {
+#if defined(SIMDE_X86_AVX512VL_NATIVE)
```

Comment:
Please be careful about indenting; the preprocessor directives should be indented in the function just like regular C code.